File: Javascript/DavidsGrade.js
0// let David's Grade be a chosen number from 1 to 10. If the grade is above 5,he is promoted. If not,he is demoted.
grade = parseInt(prompt("What is David's Grade?\n-"));
if !(grade <= 10 || grade >= 1) {
  console.error("Invalid number. Shutting down...");
}
if (grade >= 5 && grade <= 10) {
  console.log("David is promoted.");
} else if (grade < 5 && grade >= 1) {
  console.log("David is demoted.");
}

File: Python/DavidsGrade.py
# If David's Grade is below 5,he is demoted. If its 5 or above,he is promoted.
grade = int(input("What is David's Grade?\n-"))
if grade >= 11 or grade <= 0:
  print("Sorry,grade is invalid number,it must be below and above 0.")
  exit()

if grade < 5: 
  print("David is demoted.. D:")
  exit()
elif grade >= 5:
  print("David is promoted! :D")
  exit()
